ffmpeg/nodes.py

```diff
@@ -51,8 +52,21 @@ class FilterNode(Node):
     """FilterNode"""
     def _get_filter(self):
         params_text = self._name
-        arg_params = ['{}'.format(arg) for arg in self._args]
-        kwarg_params = ['{}={}'.format(k, self._kwargs[k]) for k in sorted(self._kwargs)]
+
+        # Helper function to escape uncomfortable characters
+        def escape_chars(seq, keys):
```

Comment:
Wow, I ran a few scenarios against `ffmpeg` using `drawtext`, and `ffmpeg`'s string unescaping is super bizarre. Some chars only need a single backslash before them, some require two backslashes, and some require _eight backslashes_.

Here's an example that draws the string `a\b`:
```
ffmpeg -i ffmpeg/tests/sample_data/dummy.mp4 -filter_complex '[0]drawtext=x=0:y=100:text=a\\\\\\\\b:fontfile=/Library/Fonts/Arial.ttf[1]' -map '[1]' out.mp4
```
Yup, _eight backslashes_.

I found [this stackoverflow article](https://stackoverflow.com/questions/10725225/ffmpeg-single-quote-in-drawtext) talking about it:
> Special character escapes are like violence: if they're not solving your problem, you're not using enough.
> `ffmpeg -i test.mpg -vf drawtext=text="It\\\\\'s so easy"`

I'm gonna try to see if there's any method to the madness and whether there's a way to completely correctly escape. It's nice to have this done in a tool like `ffmpeg-python` so that hopefully nobody else has to deal with ffmpeg's insanity.